Assets/Scripts/MainManager.cs

```diff
@@ -24,6 +24,11 @@ void Update()
         {
             Pause();
         }
+
+        for (int i = 0; i < GameManager.inst.GetLife(); i++)
+            hearts[i].SetActive(true);
+        for (int j = 2; j >= GameManager.inst.GetLife(); j--)
+            hearts[j].SetActive(false);
```

Comment:
```suggestion
        for (int i = 0; i < GameManager.inst.maxLife; i++)
            hearts[i].SetActive(i < GameManager.inst.GetLife());
```
저 같으면 GameManager의 maxLife를 Public으로 바꾸고 이렇게 simplify 할것 같네요~